packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,134 @@ section: components
+
+- **Toast alerts**: Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlys content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
+
+![toast notification placement](./img/toast-notification.png)
 
 ## Elements
-### Alert
-Two styles can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Toast alerts should always be dismissable, whereas inline alerts can be both dismissable and non-dismissable. Other elements are consistent between alert types.
+Toast alerts are always dismissible, and inline alerts can be both dismissable and non-dismissible. Other elements are consistent between both alert types.
 
 <img src="./img/alert-types.png" alt="Alert types" />
 
-1. **Status area:** Icon and color indicate the status or severity of the alert.
-2. **Alert title:** Include a short descriptive message. Long message text will wrap. Text color is coded to match alert type.
-3. **Close (optional):** Dismiss the alert. Required for toast alerts and optional for inline alerts.
-4. **Actions (optional):** One or more actions can be placed inside of an alert.
-5. **Description (optional):** If a longer message is required, it can be included within the Description field. This can also include a navigation link as an alternative to an action.
+1. **Status area**: The status area indicates the severity of the alert with a specific icon and color.  
 
-#### Communicating status
+2. **Alert title**: The alert title concisely communicates the alert’s main message. Its text color is coded to match the alert type. 
+
+3. **Close button**: The close button is used to dismiss the alert. It’s required for toast alerts and optional for inline alerts. 
+
+4. **Actions (optional)**: The actions area enables the user to take a specific action in relation to the alert. One or more actions can be placed inside of an alert. 
+
+5. **Description (optional)**: The description field communicates additional information to the user. It can also include a navigation link as an alternative to an action.
+
+## Communicating severity
 The status icon, background, and alert title are coded to communicate the severity of an alert.
 
-<img src="./img/alert-status.png" alt="alert styles" width="700"/>
+<img src="./img/alert-status.png" alt="alert styles" width="700" />
 
-|Type     |Icon    |Usage      |
+|Type     |Icon    |Usage      |
 |---------|--------|--------------|
-|Default  |fa-bell | Use for generic messages with no associated severity
-|Info     |fa-info-circle|Use for general informational messages (default)|
-|Warning  |fa-exclamation-triangle |Use to indicate that a non-critical error has occurred|
+|Default  |fa-bell | Use for generic messages with no associated severity
+|Information     |fa-info-circle|Use for general informational messages (default)|
```

Comment:
Just noticed that it says (default) in the information row, but then there's the default notification type which seems counterintuitive? Maybe just get rid of the "default" in parenthesis ?